### PR TITLE
Add tables, emitter, flusher for group module

### PR DESCRIPTION
**Pull request by colmazia**
## Implementation details
- Add tables in Flusher for group module
- Add hasura tracking for tables of group module
- Add emitter and flusher logic

image

## Please ensure the following requirements are met before submitting a pull request:

- [x] The pull request is targeted against the correct target branch
- [ ] The pull request is linked to an issue with appropriate discussion and an accepted design OR is linked to a spec that describes the work.
- [x] The pull request includes a description of the implementation/work done in detail.
- [ ] The pull request includes any and all appropriate unit/integration tests
- [ ] You have added a relevant changelog entry to `CHANGELOG_UNRELEASED.md`
- [x] You have re-reviewed the files affected by the pull request (e.g. using the `Files changed` tab in the Github PR explorer)

### CODE REVIEW

File: hooks/emitter/emitter.go [hunk 1]
```diff
@@ -75,11 +80,13 @@ func NewHook(
 	mintKeeper mintkeeper.Keeper,
 	distrKeeper distrkeeper.Keeper,
 	govKeeper govkeeper.Keeper,
+	groupKeeper groupkeeper.Keeper,
 	oracleKeeper oraclekeeper.Keeper,
 	icahostKeeper icahostkeeper.Keeper,
 	clientkeeper clientkeeper.Keeper,
 	connectionkeeper connectionkeeper.Keeper,
 	channelkeeper channelkeeper.Keeper,
+	groupstorekey storetypes.StoreKey,
```
Comment (RogerKSI) [inline hooks/emitter/emitter.go:89]: groupStoreKey. also, please change for ibc keeper as well.

Comment (colmazia) [reply hooks/emitter/emitter.go]: fixed

File: hooks/emitter/emitter.go [hunk 2]
```diff
@@ -101,11 +108,13 @@ func NewHook(
...
 		govKeeper:        govKeeper,
+		groupKeeper:      groupKeeper,
 		oracleKeeper:     oracleKeeper,
 		icahostKeeper:    icahostKeeper,
-		clientkeeper:     clientkeeper,
-		connectionkeeper: connectionkeeper,
-		channelkeeper:    channelkeeper,
+		clientkeeper:     clientKeeper,
+		connectionkeeper: connectionKeeper,
+		channelkeeper:    channelKeeper,
```
Comment (RogerKSI) [inline hooks/emitter/emitter.go:116]: connectionKeeper
channelKeeper
clientKeeper

**Review by RogerKSI — APPROVED** [review]

File: hooks/emitter/group.go [hunk 3]
```diff
@@ -0,0 +1,412 @@
...
+		"status":            proposalStatus,
+		"yes_vote":          tallyResult.YesCount,
+		"no_vote":           tallyResult.NoCount,
+		"no_with_veto_vote": tallyResult.NoWithVetoCount,
+		"abstain_vote":      tallyResult.AbstainCount,
+	})
+}
+
+func (h *Hook) doUpdateGroup(ctx sdk.Context, groupId uint64) {
+	groupInfoResponse, _ := h.groupKeeper.GroupInfo(
```
Comment (satawatnack) [inline hooks/emitter/group.go:281]: No error handling

Comment (RogerKSI) [reply hooks/emitter/group.go:281]: it should not happen.

**Review by satawatnack — APPROVED** [review]

**Pull request merged**